utils/build/docker/nodejs/nextjs/src/app/returnheaders/route.js

```diff
@@ -0,0 +1,6 @@
+import { NextResponse } from 'next/server'
+
+export async function GET (request) {
```

Comment:
We don't need:
```javascript
export const dynamic = 'force-dynamic'
```
here?

Reply:
nop but idk why
let me take a look

Reply:
according the docs it should be dynamic because it's using `request.headers` but it works without including `force-dynamic`

Reply:
ok, makes sense